twitter/searcher.py

```diff
@@ -168,19 +178,70 @@ def get_media_url_from_variants(variants: List[Dict[str, str]]):
 
 # TODO: hook up to s3 bucket
 def log_last_processed_id(last_processed_id: int, last_processed_time_stamp: str):
-    msg = {"last_processed_id": last_processed_id, "last_processed_time_stamp": last_processed_time_stamp}
+    msg = {
+        "last_processed_id": last_processed_id,
+        "last_processed_time_stamp": last_processed_time_stamp,
+    }
     with open(TWITTER_LAST_RUN_FILENAME, "w") as f:
         f.write(json.dumps(msg))
-    # s3_client = boto_client("s3")
-    # response = s3_client.upload_file(TWITTER_LAST_RUN_FILENAME, TWITTER_LAST_RUN_BUCKET, TWITTER_LAST_RUN_FILENAME)
-    # return response
 
 
-def get_since_id_from_file():
+def get_since_id_from_file(self):
```

Comment:
Was this supposed to change?

Reply:
Yea, makes the signature the same as the S3 one

Reply:
Would it be better to do like a **kwargs thing here? The self I just find hella confusing

Reply:
Oh yea, actually this function should be deleted

Reply:
I will delete